templates/management/website.tpl

```diff
@@ -45,7 +45,7 @@
 				{call_hook name="Template::Settings::website::appearance"}
 			</tabs>
 		</tab>
-		<tab id="setup" label="{translate key="navigation.setup"}">
+		<tab id="configuration-setup" label="{translate key="navigation.setup"}">
```

Comment:
The label for this is "Setup", so I'd rather not swap that out for a synonym ("Configuration").

There are two "setup" tabs -- one a top-level tab:
![image](https://user-images.githubusercontent.com/200411/162839947-0bf51414-eb26-48c0-8e8a-b8b9b086d5c9.png)

...and another nested inside "appearances":
![image](https://user-images.githubusercontent.com/200411/162839991-c610f66e-1ffb-4c83-9b1d-1a869937c032.png)

I'd suggest renaming the one inside "appearances" to `appearances-setup`.